microsoft-edge/webview2/concepts/versioning.md

```diff
@@ -3,15 +3,22 @@ description: Versioning Models used for Microsoft Edge WebView2
 title: Versioning of Microsoft Edge WebView2
 author: MSEdgeTeam
 ms.author: msedgedevrel
-ms.date: 10/19/2020
+ms.date: 10/28/2020
 ms.topic: conceptual
 ms.prod: microsoft-edge
 ms.technology: webview
 keywords: IWebView2, IWebView2WebView, webview2, webview, wpf apps, wpf, edge, ICoreWebView2, ICoreWebView2Host, browser control, edge html
 ---
 # Understand WebView2 SDK versions  
 
-To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload].  The minimum version that's required is included in the NuGet package version of the SDK.  For example, if you use the `SDK package version 0.9.488`, then you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload] with a build number of 488 or later.  The minimum version required is also specified in the WebView2 [Release Notes][Releasenotes].  New versions of the WebView2 SDK are shipped at the same general cadence as the Microsoft Edge \(Chromium\) browser, which is approximately every six weeks.  
+To develop a WebView2 application, you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload].  The minimum version required is specified in the WebView2 [Release Notes][Releasenotes]. For example, for SDK version [1.0.622.22](https://docs.microsoft.com/microsoft-edge/webview2/releasenotes#1062222), you must install either the [WebView2 Runtime][MicrosoftDeveloperEdgeWebview2] or a [non-stable Microsoft Edge channel][MicrosoftedgeinsiderDownload] with a build number of **86.0.616.0** or later.
+
+However, new interfaces and APIs are added in later SDK versions. If your app depends on an API added in a later version, then your app may depend on a version of the runtime greater than the SDK's minimum version.
```

Comment:
This part is a little odd and I think you want to expand a bit. I think you're trying to convey that when you have a higher build # SDK and lower build # RT, while they can be compatible and you may use all the APIs that are released prior to RT build, APIs that are released after the RT build may not be callable (giving an example here would be helpful). The best practice is to QI/try-catch (as you talked about later in the doc).

Reply:
yeah that's what I'm trying to convey

Reply:
> However, new interfaces and APIs are added in later SDK versions. 

I think this was what confused me and I had to extrapolate from what I know already :). It sounds like you are talking about us adding new functionalities to new SDKs, but you're actually referring to SDKs that have higher version number than the Runtime. Again, expanding this a little and having an example would really help here.

Reply:
@Reezaali do you have any thoughts on how to word this more clearly?

Reply:
@liminzhu is there an example you have in mind?

Reply:
pick a couple APIs and have something to the lines of - API X and Y from SDK version 200 which has a min version req 100. X was released a while ago and Y is new in SDK 200, so when you have RT version 150 (which is compatible with the SDK) X works but Y doesn't.